lib/sidekiq/middleware/current_attributes.rb

```diff
@@ -20,6 +21,16 @@ module Sidekiq
   #   Sidekiq::CurrentAttributes.persist(["Myapp::Current", "Myapp::OtherCurrent"])
   #
   module CurrentAttributes
+    class Serializer
```

Comment:
Why do you have `.to_h` at the end of the calls?

Reply:
I added the `to_h` because the `Arguments` methods return back an array of key value arrays. Like `[[:key1, :value1], [:key2, :value2]]` instead of `{key1: :value1, key2: :value2}`

Reply:
Why is that a problem?

Reply:
Might just be an issue with the tests since they deal with the internal state directly. I can just update them if that's what you'd prefer

Reply:
I figured it out. Looks like deserialization still needed to be cast because `Current.set` expects a hash